app/assets/stylesheets/front/pages/meditations.sass

```diff
@@ -203,27 +148,27 @@ body.meditations-index
 
   .slick-dots
     bottom: -10px
-    
+
     li
       button:before
         opacity: .5
         font-size: 12px
-        
+
       &.slick-active
         button:before
           opacity: 1
-          color: $blue
+          color: $color-text-blue
 
-.own-meditation
-  background: image-url("sections/try-meditation/own-meditate-full-bg.jpg") no-repeat bottom center
+.custom-meditation
+  background: image-url("meditations/background.jpg") no-repeat bottom center
```

Comment:
You were storing these images in the wrong folder.

The "Try Meditation" section is a totally different piece of content which is located on wemeditate.co/en/page/shri-mataji